js/main.js

```diff
@@ -1,60 +1,120 @@
-const form = document.querySelector('#form');
-const bookTitleField = document.querySelector('#book-title');
-const bookAuthorField = document.querySelector('#book-author');
-const booksListContainer = document.querySelector('#books-list');
-const template = document.querySelector('#list-item-template');
-const LOCAL_STORAGE_PREFIX = 'AWESOME_BOOKS';
-const BOOKS_STORAGE_KEY = `${LOCAL_STORAGE_PREFIX}-books`;
-
-function renderBookInfo(bookInfo) {
-  const templateClone = template.content.cloneNode(true);
-  const listItem = templateClone.querySelector('.list-item');
-  listItem.dataset.booksId = bookInfo.id;
-  const renderTitle = templateClone.querySelector(
-    '[data-list-item-book-title]',
-  );
-  const renderAuthor = templateClone.querySelector(
-    '[data-list-item-book-author]',
-  );
-  renderTitle.innerText = bookInfo.title;
-  renderAuthor.innerText = bookInfo.author;
-  booksListContainer.appendChild(templateClone);
+// Book Class: Represents a book
```

Comment:
- [ ] OPTIONAL
Kindly endeavor to remove **all** comments in your code hosted on the repository so as to enhance a cleaner code.